src/material.h

```diff
@@ -75,4 +77,13 @@ class material_type
         const mat_burn_data &burn_data( size_t intensity ) const;
 };
 
+namespace materials
```

Comment:
> I won't argue with that, just quote a nipped of it:

I agree it's already bad but don't make it worse by introducing another variant.

> That isn't an answer to YAGNI

> > Isn't it?

No, whats the **need**?